api/controllers/node.js

```diff
@@ -13,43 +13,54 @@ var modules;
  * @param {scope} scope - App instance.
  */
 function NodeController (scope) {
-	modules = scope.modules; 
+	modules = scope.modules;
 }
 
-NodeController.getConstants = function (req, res) {
-	modules.node.shared.getConstants(null, function (err, data){
-
-		data = _.cloneDeep(data);
-
-		//Typecast required integer or bignum attributes while sending data in API
-		data.supply = data.supply.toString();
-		data.milestone = data.milestone.toString();
-		data.reward = data.reward.toString();
-		data.fees.dappDeposit = data.fees.dappDeposit.toString();
-		data.fees.dappWithdrawal = data.fees.dappWithdrawal.toString();
-		data.fees.dappRegistration = data.fees.dappRegistration.toString();
-		data.fees.multisignature = data.fees.multisignature.toString();
-		data.fees.delegate = data.fees.delegate.toString();
-		data.fees.secondSignature = data.fees.secondSignature.toString();
-		data.fees.vote = data.fees.vote.toString();
-		data.fees.send = data.fees.send.toString();
-		data.fees.data = data.fees.data.toString();
-
-		res.send(data);
+NodeController.getConstants = function (context, next) {
+	modules.node.shared.getConstants(null, function (err, data) {
+		try {
+			if (err) { return next(err); }
+
+			data = _.cloneDeep(data);
+
+			// Typecast required integer or bignum attributes while sending data in API
```

Comment:
// Perform required typecasts for integer or bignum attributes when returning a response to the API